openapi/components/schemas/DataExports/resources/customers.yaml

```
@@ -1,4 +1,4 @@
-description: Customers resource type to export.
+description: Type of customers resource to export.
```

```suggestion
description: Customers resource to export.
```

That doesn't seem correct

---

Updated to: `Customer resource to export.`